src/components/Modal/Modal.css

```diff
@@ -0,0 +1,95 @@
+
+.modal__wrapper {
+  font-family: Aktiv Grotesk, -apple-system, BlinkMacSystemFont, San Francisco, Segoe UI, Roboto, Helvetica Neue, sans-serif;
+  position: absolute;
+  z-index: 1000;
+  width: 100%;
+  height: 100%;
+  left: 0;
+  top: 0;
+}
+
+.modal__wrapper.--hidden {
+  display: none;
+}
+
+.modal__overlay {
+  background: rgba(0, 0, 0, 0.6);
+  width: 100%;
+  height: 100%;
+  position: fixed;
+}
+
+.modal {
```

Comment:
In BEM, the main block `modal` should encompass all the elements `modal__body`, `modal__header`, `modal__footer` etc.. Here, `modal__wrapper` is written as an element of `modal` but is encompassing the `modal` inside it. I think the `wrapper` and the `overlay` are their own blocks, instead of elements of the `modal` and should be rewritten as `modal-wrapper` and `modal-overlay`.